transformer/events_creator.go

```diff
@@ -7,28 +7,26 @@ import (
 	libevents "github.com/elastic/beats/v7/libbeat/beat/events"
 	"github.com/elastic/beats/v7/libbeat/common"
 	"github.com/elastic/beats/v7/libbeat/logp"
+	"github.com/elastic/cloudbeat/evaluator"
 	"github.com/elastic/cloudbeat/resources"
 	"github.com/elastic/cloudbeat/resources/fetchers"
-	"github.com/mitchellh/mapstructure"
 	"time"
 )
 
-type CB func(ctx context.Context, input interface{}) (interface{}, error)
-
 type Transformer struct {
```

Comment:
Not for this PR.
But I am wondering if the name "Transformer" is right for this class.
It seems that this struct is responsible for the lioness part of the pipeline.
Evaluating against the evaluator and decoding the data.
So actually it is the data processor as far as I understand. 

Just a point to think about

Reply:
Thought about it as well, `processor` is taken but some variation might be good as well.